packages/client/src/__tests__/generation/generator.test.ts

```diff
@@ -43,7 +47,7 @@ describe('generator', () => {
     }
     manifest.requiresEngineVersion = 'ENGINE_VERSION_TEST'
 
-    if (process.env.PRISMA_FORCE_NAPI) {
+    if (getClientEngineType(generator.config) === ClientEngineType.NodeAPI) {
```

Comment:
Why does this get a param here, while not in many other of the usages of it?

Reply:
I think that this is not `getCliEngineType`. The one for the `client` looks into the generator

Reply:
Removed as it is not necessary here.